src/functionalTest/kotlin/com/figure/gradle/semver/gradle/runner.kt

```diff
@@ -52,7 +52,6 @@ fun runner(
         "--build-cache",
         "--configuration-cache",
         "--stacktrace",
-        "-Psemver.forTesting=true",
```

Comment:
No need for this now that I found that I can mutate the environment variables for testing with `withEnvironment` and `SystemEnvironmentProjectListener` via Kotest